modules/python/Makefile

```diff
@@ -10,22 +10,36 @@ python/check: python/fmt python/lint python/test
 
 ## Lint Python code
 python/lint: python/typecheck
+	@echo "${GREEN}Running flake8 lint checks${RESET}"
 	@flake8 $(PY_FILES)
 	@isort --check-only --diff $(PACKAGE)
 
 ## Typecheck Python code
 python/typecheck:
+	@echo "${GREEN}Running mypy typing checks${RESET}"
 	@mypy $(PY_FILES)
 
 ## Run Python unit tests
 python/test:
+	@echo "${GREEN}Running python tests${RESET}"
 	@pytest $(PYTEST_FILES)
 
 ## Format Python code
 python/fmt:
+	@echo "${GREEN}Running isort auto-format${RESET}"
 	@isort $(PY_FILES)
+	@echo done
+	@echo "${GREEN}Running black auto-format${RESET}"
 	@black $(PY_FILES)
 
+## Format Python code
+python/check-fmt:
+	@echo "${GREEN}Running isort format checks${RESET}"
+	@isort --check $(PY_FILES)
+	@echo done
+	@echo "${GREEN}Running black format checks${RESET}"
+	@black --check $(PY_FILES)
```

Comment:
The `--check` flag makes the programs exit with non-zero status if there are formatting errors.